src/Controls/src/Core/SemanticProperties.cs

```diff
@@ -47,10 +47,12 @@ public static void SetHeadingLevel(BindableObject bindable, SemanticHeadingLevel
 			SemanticProperties.DescriptionProperty,
 			SemanticProperties.HintProperty,
 			SemanticProperties.HeadingLevelProperty,
+#pragma warning disable CS0618 // Type or member is obsolete
 			AutomationProperties.NameProperty,
 			AutomationProperties.LabeledByProperty,
 			AutomationProperties.HelpTextProperty,
 			AutomationProperties.IsInAccessibleTreeProperty,
```

Comment:
let's keep the IsInAccessibleTreeProperty for now!

Reply:
Oops, fixed thanks! Not that this should impact anything besides disabling the warnings in the IDE but better keep things in order!